essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,2 +1,200 @@
+
+### General Purpose of this Article
+<p>We will explore the ins and outs of contributing to Appflowy and provide you with practical tips on how to make your first contribution. We'll demystify the process, highlight the benefits, and equip you with the essential tools and resources to get started on your open-source journey with Appflowy.
+
+Throughout this guide, we'll address common questions you may have, such as:
+
+1. How to contribute to Appflowy?
+2. How to connect with the team?
+3. How to identify and solve your first Pull Request?
+4. How to propose new features for Appflowy?
+5. How to best engage with the community?
+  
+And many more. Let's dive in!</p>
+<p align="right">(<a href="#introduction">back to top</a>)</p>
+
+
+<!-- How to Contribute -->
+## How to Contribute to Appflowy
+<p>Appflowy is managed via GitHub, a popular Content Management/Version Control Application. You'll need a GitHub account to get started for the same. We recommend getting used to GitHub's features and functions, especially Git and GitHub Workflows, before starting with contributions.</p>
+
+### How to Connect with the Team
+<p>The best ways to connect with the team are via Discord and GitHub. </p> 
+
+1. Discord 
+  - Create a Discord account, and join the [Appflowy Community Server](https://discord.com/invite/9Q2xaN37tV). The server is full of creative individuals from all over the world, who are always ready to help you out, so do not be afraid to ask questions!
+  - Start of by introducing yourself in the <b>Intro's</b> section. Read carefully through the <b>welcome</b> Section to better abide by the server's rules and regulations.
+  - Have a look around! Feel free to ask any prevalent questions to any member of the Appflowy Team or Appflowy Contributor's by sending a message with a tag (<b>@AppFlowy Team </b>) for the same.
+
+![image](https://user-images.githubusercontent.com/70965472/237025457-bf33e70d-8fbd-4bea-a25e-f1d0ea2147bf.png)
+*Welcome Page - Appflowy Dicord Server*
+
+2. GitHub  
+  - You can get started via asking questions in the [GitHub Discussions](https://github.com/AppFlowy-IO/AppFlowy/discussions) section of Appflowy.
+  - GitHub Discussions provides a forum for Appflowy Contributor's to Ask questions, Share ideas, Engage with other community members and to subsequently Welcome others who join in down the road.
+
+![image](https://user-images.githubusercontent.com/70965472/237032567-9d75b320-423e-4bc2-805d-7bcb7c96217c.png)
+*The Appflowy Discussions page*
+
+   - You can navigate through the various categories made available, including announcements, ideas, polls, etc.
```

Comment:
Is this specific to appflowy? If not, can we point them to a guide instead?

Reply:
This is specific to the GitHub Discussions page for Appflowy's main repository. I have added links to the Appflowy docs below, felt that it would be best to include it here as another means to connect with the community

Reply:
Sounds good to me!